File: OpenWeatherWebApp/src/app/shared/DTOs/UnitsDTO.ts
export class UnitsDTO
{
    constructor(
        public tempDefault:number, 
        public tempMetrics:number, 
        public tempImperial:number, 
        public tempMinDefault:number, 
        public tempMinMetrics:number, 
        public tempMinImperial:number, 
        public tempMaxDefault:number, 
        public tempMaxMetrics:number, 
        public tempMaxImperial:number,
        public feelsLikeDefault:number, 
        public feelsLikeMetrics:number, 
        public feelsLikeImperial:number
    ) {}
}

File: OpenWeatherAPI/OpenWeatherAPI.Models/Models/UnitsDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OpenWeatherAPI.Models.Models
{
    public class UnitsDTO
    {
        public double TempDefault { get; set; }
        public double TempMetrics { get; set; }
        public double TempImperial { get; set; }
        public double TempMinDefault { get; set; }
        public double TempMinMetrics { get; set; }
        public double TempMinImperial { get; set; }
        public double TempMaxDefault { get; set; }
        public double TempMaxMetrics { get; set; }
        public double TempMaxImperial { get; set; }
        public double FeelsLikeDefault { get; set; }
        public double FeelsLikeMetrics { get; set; }
        public double FeelsLikeImperial { get; set; }

    }
}
